portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ABTestDXPSide.testcase

```diff
@@ -330,23 +319,9 @@ definition {
 
 		task ("Assert the variant published") {
 			ContentPages.viewFragmentText(
-				fragmentName = "heading",
-				id = "element-text",
-				text = "Heading Example");
-		}
-
-		task ("Now edit the variant and add a widget to the page and then click save") {
-			ContentPages.gotoPageEditor();
-
-			PageEditor.addFragment(
-				collectionName = "Basic Components",
-				fragmentName = "Button");
-
-			PageEditor.publish();
-		}
-
-		task ("Assert if the page is published with the modifications") {
-			PageEditor.viewFragment(fragmentName = "Button");
```

Comment:
Hey @joshitagaki, this edit that the test made to the page after the variant was published was to ensure that [this bug](https://liferay.atlassian.net/browse/LRAC-14634) would not occur. I think it would be important to keep this checking. You can change the steps to change just the button text as you did previously which is enough to guarantee this. What do you think?